### ISSUE TITLE
module version mismatch?

### ISSUE THREAD

**Issue by asadhash** [tjanczuk/edge#276]
Hi Tomasz,

I'm a newbie. I installed the following without errors.. :-)
    node.js v0.12.2
    edge@0.10.0
    edge-sql@0.1.0
    express@4.12.3

I got a sigsegv error with mono everywhere. Am I installing the correct module versions?
Would really appreciate any help. 

I also posted this to stackoverflow 
http://stackoverflow.com/questions/30093491/sigsegv-error-after-installing-node-js-express-and-edge-mismatch-of-versions-o

**Comment by tjanczuk** [comment]
What is your OS and how did you install edge?

**Comment by asadhash** [comment]
Hey Tomasz,

Super cool that you're taking the time to help. Really really appreciate it. 

I installed it on my Apple MacBook Pro Yosemite 10.10.4

You have a set of instructions on your website. I followed those to install edge. https://github.com/tjanczuk/edge/tree/master#building-on-osx

Edge installed ok. I think it might have to do with my code. Or some module version mismatch. 

Warm regards
--
Arun Sadhashivan | +31614552998
